src/N64API.hpp

```diff
@@ -154,4 +156,55 @@ N64_Data_t CN64Controller::getData(void)
 // N64 Console API
 //================================================================================
 
-// TODO
+CN64Console::CN64Console(const uint8_t p) : pin(p){
+    // Empty
+}
+
+
+bool CN64Console::write(N64_Data_t &data)
+{
+    // Abort if controller was not initialized.
+    // N64 will refuse and weird connect/disconnect errors will occur.
+    //if (data.report.origin) {
```

Comment:
Why did you disable this?

Reply:
N64 doesn't have a special origin report, it just has the initial status request command (0x00) and the report request/input poll (0x01). Sorry about leaving it commented out, I forgot to remove that. I'll delete it now

Reply:
That makes sense. It should be removed then :-)